bin/stress-test/README.md

```diff
@@ -138,5 +139,12 @@ P99.9 request latency: 2.289709ms
 Average nullifiers per response: 21.0348
 ```
 
+- load-state
+``` bash
+$ miden-node-stress-test benchmark-store --data-directory ./data load-state
+
+State loaded in 41.494029542s
+```
```

Comment:
It would be great to make it clear how big the underlying state is. On line 71 we say that the DB has about 100K accounts and 100K notes. On line 45 we say that there are 1M accounts (which I think is incorrect). And I don't think we ever say how many nullifiers are in the store.

It may make sense to have a small paragraph at the beginning of the "Results" section to explain the dataset which will be used for all benchmarks.

------

Separately, if we turn on tracing, could we get info on how long it takes to load the MMR, account tree, and nullifier tree separately? Would be great to put this into a table.

Reply:
Looking at the tracing results, we basically have:

- Account and nullifier tree take about 21 seconds to load each.
- Everything else (including the MMR loading) is basically negligible.

I would probably add a comment about this somewhere next to the benchmark.